packages/react-components/react-tooltip/src/components/Tooltip/Tooltip.tsx

```diff
@@ -4,6 +4,7 @@ import { renderTooltip_unstable } from './renderTooltip';
 import { useTooltipStyles_unstable } from './useTooltipStyles';
 import type { TooltipProps } from './Tooltip.types';
 import type { FluentTriggerComponent } from '@fluentui/react-utilities';
+import { useCustomStyleHooks_unstable } from '@fluentui/react-shared-contexts';
```

Comment:
_nit:_ Keep `import type` statements at the bottom

```suggestion
import { useCustomStyleHooks_unstable } from '@fluentui/react-shared-contexts';
import { useTooltipStyles_unstable } from './useTooltipStyles';
import type { TooltipProps } from './Tooltip.types';
import type { FluentTriggerComponent } from '@fluentui/react-utilities';
```